src/sentry/api/endpoints/organization_events.py

```diff
@@ -22,10 +22,13 @@ def get_project_ids(self, request, organization):
 
         requested_projects = project_ids.copy()
 
-        om_role = OrganizationMember.objects.filter(
-            user=request.user,
-            organization=organization,
-        ).values_list('role', flat=True).get()
+        try:
+            om_role = OrganizationMember.objects.filter(
+                user=request.user,
+                organization=organization,
+            ).values_list('role', flat=True).get()
```

Comment:
does .first() work on this kind of queryset? if so it lets you simplify this code mostly to what it was before

Reply:
Please no, `.first()` adds an implicit `ORDER BY id` which changes the query plans sometimes into something bad and unintended.

Reply:
We can just override the manager but using first reduces a lot of boilerplate code.